lib/feature/detail_card/detail_card_bloc.dart

```diff
@@ -81,11 +85,12 @@ class DetailCardBloc extends BlocBase {
 
   final isLoadingUpdateSubject = BehaviorSubject<bool>.seeded(false);
 
-  final titleController = TextEditingController();
-
   final focusNodeTitle = FocusNode();
 
   final isShowChecklistSubject = BehaviorSubject<bool>.seeded(false);
+  final isLoadingAddCommentSubject = BehaviorSubject<bool>.seeded(false);
+
+  final dataKey = GlobalKey();
```

Comment:
```suggestion
  final dataKey = GlobalKey();
```
nên cho cái key này vào đây
![image](https://github.com/Capstone-Project-Fpoly/task-manager-app/assets/102580970/ba9db509-9832-41b2-85d8-444bd1fb4232)